presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx

```diff
@@ -241,7 +241,7 @@ const HISTOGRAM_PROPERTIES = {
     disableHiddenCheck: true,
 };
 
-class StageDetail extends React.Component {
+class StageSummary extends React.Component {
```

Comment:
Is this a preexisting issue? If so, can you add details in the commit explaining the impact of this fix? (i.e., was part of the UI broken?). If not, it should be squashed with the relevant commit.